lib/signet/oauth_2/client.rb

```diff
@@ -167,6 +170,8 @@ def initialize options = {}
       #   - <code>:extension_parameters</code> -
       #     When using an extension grant type, this is the set of parameters used
       #     by that extension.
+      #   - <code>:granted_scopes</code> -
```

Comment:
Can we use backticks instead of html tags?

Reply:
Done